### PR TITLE
Updated home page.

### PR DESCRIPTION
**Pull request by fishscene**
Added direct links to Linux/Mac/Windows installers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +13,25 @@ Streamlink is a fork of the [livestreamer](https://github.com/chrippa/livestream
 
 Streamlink can be installed either via: `pip` or manually via `python setup.py install`.
 
-Pip:
+**Python Pip:**
```
Comment (gravyboat) [inline README.md]: Could we do `Python via pip` here so it's a complete phrase? Also no need to capitalize pip.

**Comment by gravyboat** [comment]
Looks good! I added one note but past that this is great, thank you!

**Comment by beardypig** [comment]
Looking good! I agree with @gravyboat.

**Comment by fishscene** [comment]
I should be able to make this change tonight. Should I make a new pull request? I'm not sure how to modify an existing PR - or if that's even possible.

**Comment by beardypig** [comment]
@fishscene normally you can add commits to a PR by just pushing more commits, but I'm not sure you can if you make the PR from your master branch. The best way to do it is create a new branch, make the changes to that branch, then push the branch to your fork, and use that new branch for the PR. Then, if you need to make changes you can commit to that branch and push to your fork :-)

**Comment by gravyboat** [comment]
These changes look good, thanks @fishscene!

**Pull request merged**